packages/wonder-blocks-birthday-picker/src/components/birthday-picker.tsx

```diff
@@ -345,7 +375,7 @@ export default class BirthdayPicker extends React.Component<Props, State> {
                 placeholder={this.labels.year}
                 onChange={this.handleYearChange}
                 selectedValue={year}
-                style={{minWidth, ...dropdownStyle}}
+                style={{minWidth, ...defaultStyles.input, ...dropdownStyle}}
```

Comment:
**suggestion:** This is kinda unrelated to your changes (some previous changes started this), but I think it would be better keeping the array structure to avoid possible merge style conflicts.

```suggestion
                style={[{minWidth}, defaultStyles.input, dropdownStyle]}
```